mantis-runtime/src/main/java/io/mantisrx/runtime/GroupToScalar.java

```diff
@@ -126,6 +127,11 @@ public Config<K, T, R> concurrentInput(final int concurrency) {
             return this;
         }
 
+        public Config<K, T, R> shouldUseProactiveRouter(boolean useProactiveRouter) {
```

Comment:
shall these names be consistent across stage config? e.g withProactiveRouter in keytokey.